bokeh/models/renderers.py

```diff
@@ -52,6 +51,10 @@ class TileRenderer(Renderer):
     Specifies the level in which to render the glyph.
     """)
 
+    render_parents = Bool(default=True, help="""
```

Comment:
Will `render_parents` attribute be implemented in other descendants of `Renderer` class, other than `TileRenderer`? If not, then this should be moved to appropriate subclass (`TileRenderer`).